### PR TITLE
Update faq

### PR DESCRIPTION
**Pull request by amuntner**
### Guidelines
Before you submit a pull request, please make sure you have done the following:

- [ ] Review the guidelines for contributing to this repository (link above in GitHub ^).
- [ ] All commit messages should be well written, and reference a Bugzilla bug number where applicable.

Note: If you're unsure about anything here or below, don't worry! Just fill out as much
information as you can.

## Description
Update program information

## Bugzilla
N/A

## Testing
n/a

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

**Comment by pmclanahan** [comment]
This looks pretty good to me. I'd like to have a more descriptive commit message though.

@craigcook markup look good to you?

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html [hunk 1]
```diff
@@ -81,19 +81,14 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
             encourage people to examine typical areas for vulnerabilities such as
             authentication and session management. Since our code is opensource, you are
             encourage to run on the software on your own server instance or just look
-            at the source code for potential issues.
-            (See the list below for the domains and applications which are in scope.)</p>
+            at the source code for potential issues.</p>
+            <p>On Mozilla's Wiki, you can find information about some of our <a href="https://wiki.mozilla.org/Security/TestingToolchains/">security testing toolchains. </a>
```
Comment (craigcook) [inline bedrock/security/templates/security/bug-bounty/faq-webapp.html]: Should be a `</p>` to close this paragraph before the next one opens.

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html [hunk 2]
```diff
@@ -81,19 +81,14 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
...
+            <p></p>See the list below for the domains and applications which are in scope.</p>
```
Comment (craigcook) [inline bedrock/security/templates/security/bug-bounty/faq-webapp.html]: Premature `</p>` here.

File: bedrock/security/templates/security/bug-bounty/faq-webapp.html [hunk 3]
```diff
@@ -179,21 +179,14 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
...
-          <p>We also ask that you notify the <a
-            href="mailto:security@mozilla.org?subject=Web%20Security%20Bug%20Bounty:&lt;
-  DESCRIPTION&gt;">
-            Mozilla Security Group</a> by email and include the bug number and a brief
-            summary.
+          <p>Please <a href="https://bugzilla.mozilla.org/form.web.bounty">
+            file a bug</a>.
+            When filing, please verify that you have followed the guidance on how to write your report in order to assure it is evaluated as quickly as possible. 
+            Repeat the attack using only your own description in order to prevent errors and omissions, then update your documentation to be submitted.</p>
+            <p><b>Please do not ever submit website vulnerabilities via email.<b>
```
Comment (craigcook) [inline bedrock/security/templates/security/bug-bounty/faq-webapp.html]: This should be `<strong>` rather than `<b>`

**Comment by craigcook** [comment]
Some small things but no major issues here either. The formatting could be tidied up a bit and long lines should be wrapped consistently with the rest of the document but that's nitpicky, I can look the other way just this once ;)

**Comment by craigcook** [comment]
A made a few markup and formatting tweaks and merged this in https://github.com/mozilla/bedrock/commit/27b7e387bcf7247829eaa9a416aea37e4a0f8845

**Pull request closed without merging**